code.rb

```diff
@@ -0,0 +1,56 @@
+module Enumerable
+  def my_each
+    for i in 0...self.length
+      yield (self[i])
```

Comment:
Let's remove the space beyween `yesild` and the parenthesis.

```suggestion
      yield(self[i])
```